graylog2-web-interface/src/views/components/sidebar/AddWidgetButton.jsx

```diff
@@ -2,11 +2,30 @@
 import * as React from 'react';
 import uuid from 'uuid/v4';
 import { PluginStore } from 'graylog-web-plugin/plugin';
+import styled from 'styled-components';
+import { sortBy, isEmpty } from 'lodash';
 
-import { ButtonGroup, Button } from 'components/graylog';
+import { Button } from 'components/graylog';
 import { ViewStore } from 'views/stores/ViewStore';
 import View from 'views/logic/views/View';
 
+
+const Group = styled.div`
+  margin-bottom: 20px;
+  &:last-child {
```

Comment:
the `&` is unnecessary unless it helps you read the code better.
Also, what do you think about having a blank line between blocks of styles?

Reply:
Good to know. I like the idea of a blank line between blocks of styles.